src/main/java/com/jcabi/github/Issue.java

```diff
@@ -120,6 +120,18 @@
      */
     boolean exists() throws IOException;
 
+    /**
```

Comment:
@paulodamaso How exactly can a user react to an ``Issue``? I thought reactions are only for comments :D

Reply:
@amihaiemil Me too, but ... https://developer.github.com/v3/reactions/#create-reaction-for-an-issue